copyhtml/prompt.ts

```diff
@@ -0,0 +1,18 @@
+export interface PromptResult {
+  selectors: string[];
+}
+
+const PROMPT = "Enter selectors to copy HTML from (comma-separated):";
+
+export function getPromptResult(): PromptResult {
+  const commaSeparatedSelectors = prompt(PROMPT);
```

Comment:
Utilize [`prompt`](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt#parameters)'s second argument, `defaultValue` to not overwrite the user's clipboard unless selectors are passed via the prompt.

Reply:
Resolved by <https://github.com/EthanThatOneKid/copyhtml/pull/1/commits/4cc25a7b54ce6437b7b9b6b2d1fc42942e347a96>.